xain_fl/coordinator/coordinator.py

```diff
@@ -112,9 +109,9 @@ class Coordinator:  # pylint: disable=too-many-instance-attributes
 
     def __init__(  # pylint: disable=too-many-arguments
```

Comment:
please have a look at https://github.com/xainag/xain-sdk/pull/88#discussion_r380557143 about writing of initial weights.

Reply:
Yep that's done: https://github.com/xainag/xain-fl/pull/298/commits/f1efdc6d375022c7b6911857ad34c73934ac22d5